packages/inquirer/lib/objects/choices.js

```diff
@@ -1,6 +1,5 @@
 import assert from 'node:assert';
-import filter from 'lodash/filter.js';
-import map from 'lodash/map.js';
+import filter from 'lodash.filter';
```

Comment:
Do we need filter from lodash?

Reply:
the `where` method we have is relying on the object syntax it supports afaict

i.e. `filter(arr, {someKey: 'someValue'})` rather than `arr.filter((v) => v.someKey === 'someValue')`